src/Symfony/Component/Security/Core/Encoder/NativePasswordEncoder.php

```diff
@@ -27,7 +27,10 @@ final class NativePasswordEncoder implements PasswordEncoderInterface, SelfSalti
     private $algo;
     private $options;
 
-    public function __construct(int $opsLimit = null, int $memLimit = null, int $cost = null)
+    /**
+     * @param int|string|null $algo An algorithm supported by password_hash(). If null, the stronger available algorithm will be used.
```

Comment:
let's make it `string|null`: this is future proof, and numbers will be cast for us so it should work seamlessly.